src/constants.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
   provideTimeout: 15 * SECOND,
   kMaxPriority: Math.pow(2, 31) - 1,
   rebroadcastDelay: 10 * SECOND,
-  maxListeners: 1000
+  maxListeners: 1000,
+  wantlistSendDebounceAmount: 10
```

Comment:
Nit: can we include the unit in the variable name, eg `wantlistSendDebounceMs`

Reply:
Sure, but none of the other have units.